androguard/core/apk/__init__.py

```diff
@@ -1506,6 +1506,10 @@ def get_certificate_der(self, filename:str) -> bytes:
         pkcs7message = self.get_file(filename)
 
         pkcs7obj = cms.ContentInfo.load(pkcs7message)
+        # TODO: should be returning the matching cert here!
+        # https://github.com/androguard/androguard/pull/1038
+        if len(pkcs7obj['content']['certificates']) > 1:
+            logger.warning(f"Multiple certificates found. Returning the first one!")
```

Comment:
FYI: this f-string doesn't have any replacement fields (I'd expect a linter to catch that).

Reply:
thanks, initially i had it returning the number of certificates, but it did not add much of a value so I removed it.